app/policies/target_policy.rb

```diff
@@ -6,4 +6,8 @@ def create?
   def index?
     true
   end
+
+  def destroy?
+    true
```

Comment:
I think this is insecure because you are authorizing any user to delete any target. Maybe we should check that the user is authenticated (`current_user.present?`) and also check that the Target belongs to this user

Reply:
Done! I have done some changes in the policy in case you want to check them, thanks!

Reply:
take a look at this file `user_policy_spec.rb` is good example

Reply:
Done! I just added the target_policy_spec.rb, let me know if like this is OK or i have to change/add something else. Thanks!